### PR TITLE
Upgraded Lucene in Big-Data-Utils

### LINKED ISSUE #239: Upgrade the dependencies of big-data-utils
Upgrade the dependencies of big-data-utils which is introduced in #119 : 

Quote from Mark: 

> This module uses a very old -- and unsupported -- version of mapdb; http://www.mapdb.org/doc/ and Lucene: https://lucene.apache.org/. It may be worth updating to newer versions

### PR DESCRIPTION
**Pull request by shyamupa**
Now works with Lucene 6.0.0. 

Closes #239

### CODE REVIEW

File: big-data-utils/src/main/java/edu/illinois/cs/cogcomp/bigdata/lucene/Lucene.java [hunk 1]
```diff
@@ -267,9 +266,9 @@ public static QueryParser newQueryParser(String string) {
 	 * @param id
 	 * @return
 	 */
-	public static Query intQuery(String field, int id) {
-		return NumericRangeQuery.newIntRange(field, id, id, true, true);
-	}
+//	public static Query intQuery(String field, int id) {
```
Comment (mssammon) [inline big-data-utils/src/main/java/edu/illinois/cs/cogcomp/bigdata/lucene/Lucene.java]: Is this no longer needed?

File: tokenizer/pom.xml [hunk 2]
```diff
@@ -15,17 +15,17 @@
         <dependency>
             <groupId>edu.illinois.cs.cogcomp</groupId>
             <artifactId>illinois-core-utilities</artifactId>
-            <version>3.1.12</version>
+            <version>3.1.13</version>
         </dependency>
         <dependency>
             <groupId>edu.illinois.cs.cogcomp</groupId>
             <artifactId>LBJava-NLP-tools</artifactId>
-            <version>3.1.12</version>
```
Comment (mssammon) [inline tokenizer/pom.xml]: not sure why all these pom version changes show up; in general, we leave the version as is, and it gets incremented by a deploy build operation on semaphore/jenkins.  Please reset to whatever the current version is.

Comment (shyamupa) [inline tokenizer/pom.xml]: Ah, I used the increment pom script. Will revert this.

File: big-data-utils/src/main/java/edu/illinois/cs/cogcomp/bigdata/lucene/Lucene.java (hunk 1 above)
Comment (shyamupa) [inline big-data-utils/src/main/java/edu/illinois/cs/cogcomp/bigdata/lucene/Lucene.java]: Lucene has removed this kind of queries from the API. All queries are to be made using the QueryParser now.

**Comment by danyaljj** [comment]
@shyamupa could you resolve the conflicts and merge it?

**Pull request merged**